File: src/VSServiceStack/ProjectTemplates/FSharpAngular/FSharpAngular/AppHost.fs
namespace $safeprojectname$
    open ServiceStack
    open ServiceStack.Common
    open ServiceStack.Razor
    open $safeprojectname$
    open $safeprojectname$.ServiceInterface
    open System

    type AppHost() = 
        inherit AppHostBase("$safeprojectname$", typeof<MyServices>.Assembly)
        override this.Configure container =
            //Add plugins
            //this.Plugins.Add(new PostmanFeature())
            //this.Plugins.Add(new CorsFeature())
            this.Plugins.Add(new RazorFormat())
            ignore()
        static member Start() = 
            new AppHost() |> (fun x -> x.Init()) |> ignore

File: src/ServiceStackVS/ProjectTemplates/Bootstrap/ServiceStackBootstrap/AppHost.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using Funq;
using ServiceStack;
using ServiceStack.Razor;
using $saferootprojectname$.ServiceInterface;

namespace $safeprojectname$
{
    public class AppHost : AppHostBase
    {
        /// <summary>
        /// Base constructor requires a Name and Assembly where web service implementation is located
        /// </summary>
        public AppHost()
            : base("$safeprojectname$", typeof(MyServices).Assembly) {}

        /// <summary>
        /// Application specific configuration
        /// This method should initialize any IoC resources utilized by your web service classes.
        /// </summary>
        public override void Configure(Container container)
        {
            //Config examples
            //this.Plugins.Add(new PostmanFeature());
            //this.Plugins.Add(new CorsFeature());
            
            this.Plugins.Add(new RazorFormat());
        }
    }
}